integration_tests/__init__.py

```diff
@@ -78,8 +78,10 @@ def get_output_ignoring_non_zero_exit(self, binary, cwd):
             output = exception.output
         return output
 
-    def login(self, email='u1test+snapcraft@canonical.com',
+    def login(self, email=None,
               password=None, expect_success=True):
+        email = email or os.getenv('TEST_USER_EMAIL',
+                                   'u1test+snapcraft@canonical.com')
```

Comment:
De nada but what are you thanking me for ? Using the same name default account for tests ?